src/validation/addComment.ts

```diff
@@ -0,0 +1,13 @@
+import { IComment } from '../interfaces/IComment'
+import joi from 'joi'
+
+const validateComment = (data: IComment) => {
+  const imagePattern = /\.(jpe?g|png|gif|bmp)$/i
+  const schema = joi.object({
+    image: joi.string().pattern(imagePattern),
```

Comment:
Maybe better if you use it as uri
```suggestion
    image: joi.string().uri().pattern(imagePattern),
```